pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[project]
+name = "pebbleos"
+version = "0.1.0"
+description = "Codebase for firmware that runs on Pebble smartwatches"
+readme = "README.md"
+requires-python = ">=3.13"
+dependencies = [
+    "bitarray>=3.5.1",
+    "boto>=2.49.0",
+    "certifi>=2025.7.14",
+    "cobs==1.0.0",
+    "enum34>=1.1.10",
+    "freetype-py>=2.5.1",
+    "gitpython==1.0.1",
+    "grpcio-tools>=1.73.1",
+    "intelhex>=2.1,<3",
+    "libclang>=18.1.1",
+    "libpebble2>=0.0.16",
+    "mock>=5.2.0",
+    "myst-parser~=4.0",
+    "nanopb>=0.4.9.1",
+    "nose>=1.3.7",
+    "packaging>=25.0",
+    "pathlib>=1.0.1",
+    "pebble-commander",
+    "pebble-loghash",
+    "pebble-programmer",
+    "pebble-pulse2",
+    "pep8>=1.7.1",
+    "pexpect>=4.9.0",
+    "pillow>=11.3.0",
+    "ply==3.4",
+    "polib>=1.2.0",
+    "prompt-toolkit>=0.55",
+    "protobuf>=6.31.1",
+    "pycryptodome>=3.23.0",
+    "pyelftools>=0.32",
+    "pyftdi==0.56.0",
+    "pypng>=0.20220715.0",
+    "pyserial>=3.5",
+    "pyusb==1.3.1",
+    "requests>=2.32.4",
+    "sh==1.4",
+    "sphinx~=8.2",
+    "sphinx-book-theme~=1.1",
+    "sphinx-design~=0.6",
+    "svg-path>=7.0",
+]
+
+[tool.uv.sources]
```

Comment:
isn't there any standard way to define this? it worked pre-uv...

Reply:
[Looks like there is](https://stackoverflow.com/a/77181715). I'll do that instead.

Reply:
Actually, nope, I've run into astral-sh/uv#3271 - doesn't work if you run uv outside of the repo root (eg. in `docs/`). Will probably have to put that one back, unless there's any better way to do it.